packages/ui-components/package.json

```diff
@@ -6,11 +6,14 @@
     "type": "git",
     "url": "https://github.com/cockroachdb/ui"
   },
-  "main": "dist/main.js",
+  "source": "src/index.ts",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
```

Comment:
so uh this was all the config necessary for parcel, besides the `.sassrc.json` file. It's weird, I know.

Reply:
What is the `sassrc` file for? Is it just telling parcel that we are using sass?

Reply:
Parcel automatically detects sass based on our `.ts` files importing a `.scss` file! The `.sassrc.json` fiel is mostly to make it explicit that `sass` should import packages from `node_modules`. I'll try to remove it — may not be necessary anymore.

Reply:
Huh, that's interesting! Thanks for the explanation.